ui/src/components/DisplayAmount.tsx

```diff
@@ -33,7 +33,7 @@ export const AmountValue = ({
   displayInfo,
   allowPopup,
 }: {
-  amount: Amount;
+  amount: Amount<'set'> | Amount<'nat'> | Amount<'copyBag'> | Amount<'copySet'>;
```

Comment:
can it really display all those amounts?

Reply:
on a second look `Amount<'set'>['value'] | Amount<'copyBag'>['value']` is all it needs. I'll adjust accordingly.